src/rendition-mixin.js

```diff
@@ -92,6 +92,9 @@ let renditionSelectionMixin = function(hlsHandler) {
 
   // Add a single API-specific function to the HlsHandler instance
   hlsHandler.representations = () => {
+    if (!playlists || !playlists.master || !playlists.master.playlists) {
```

Comment:
We should probably return an empty array if we don't have playlists yet, Right now we will just throw an error and some of the tests where failing due to this.